src/stackdriver-nozzle/nozzle/nozzle.go

```diff
@@ -0,0 +1,21 @@
+package nozzle
+
+import (
+	"github.com/evandbrown/gcp-tools-boshrelease/src/stackdriver-nozzle/stackdriver"
+	"fmt"
+)
+
+
+type Nozzle struct {
+	StackdriverClient stackdriver.Client
+}
+
+func (n *Nozzle) Connect() bool {
+	return true
+}
+
+func (n *Nozzle) ShipEvents(event map[string]interface{}, _ string /* TODO research second string */) {
```

Comment:
The interface specifies the second string - it turns out to be a log message, but we haven't implemented any events that actually use it yet.